spec/controllers/two_factor_authentication/personal_key_verification_controller_spec.rb

```diff
@@ -86,7 +86,6 @@
       profile = create(:profile, :active, :verified, pii: { ssn: '1234' })
       user = profile.user
       raw_key = PersonalKeyGenerator.new(user).create
-      old_key = user.reload.encrypted_recovery_code_digest
```

Comment:
I don't suppose the `.reload` was meaningful for anything after the assignment? I guess if it passes, it passes 🤷